docker-compose.yml

```diff
@@ -1,6 +1,13 @@
 version: '2'
 
 services:
+  webapp:
+    build: .
+    ports:
+      - "8080:8080"
+    env_file:
+      - webapp.env
```

Comment:
I think the [mongo container's setup](https://github.com/helloGov/webapp/blob/master/scripts/provision_mongo.sh#L2) points to `.env`. Do both containers use the same file?